pyintegration/tests/test_produce_kafka.py

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import os
+import unittest
+
+from deephaven2 import kafka_producer as pk, new_table
+from deephaven2.column import string_col, int_col, double_col
+from tests.testbase import BaseTestCase
+
+
+def table_helper():
+    columns = [
+        string_col('Symbol', ['MSFT', 'GOOG', 'AAPL', 'AAPL']),
+        string_col('Side', ['B', 'B', 'S', 'B']),
+        int_col('Qty', [200, 100, 300, 50]),
+        double_col('Price', [210.0, 310.5, 411.0, 411.5])
+    ]
+    t = new_table(cols=columns)
+    return t
+
+
+class KafkaProducerTestCase(BaseTestCase):
+    """
+    Test cases for the deephaven.ConsumeKafka  module (performed locally) -
+    """
+
+    def test_basic_constants(self):
+        """
+        Check that the basic constants are imported and visible.
+        """
+        self.assertIsNotNone(pk.IGNORE)
+
+    def test_simple(self):
+        """
+        Check a simple Kafka subscription creates the right table.
+        """
+        t = new_table(cols=[double_col('Price', [10.0, 10.5, 11.0, 11.5])])
+        cleanup = pk.produce(
+            t,
+            {'bootstrap.servers': 'redpanda:29092'},
+            'orders',
+            key=pk.IGNORE,
```

Comment:
Looking at this, I think an `ignore()` function may be the way to go.  That way, a user knows they always need to call a function, instead of switching between a function and a constant.